en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -1,94 +1,121 @@
 
 1. Expose an existing API by creating a REST API proxy
 2. Deploy the REST API proxy
 3. Test the REST API proxy to verify its functionality
-4. Manage your REST API proxy by adding rate limiting and security policies and leveraging the platform's API management capabilities.
+4. Manage your REST API proxy by adding rate limiting and security policies, thereby leveraging the platform's API management capabilities.
 
 !!!tip "Before you begin..."
     Familiarize yourself with REST API Proxies in Choreo. [REST API Proxies](../../develop/components/api-proxies/api-proxy.md) 
 
 
-Let's begin...
+Let's begin!
+
 ## Step 1: Create a REST API Proxy
 
 You can create a REST API proxy in one of the following methods: 
 
-- Design a new REST API proxy using an existing endpoint. 
-- Importing an OpenAPI specification of the existing API.
 
-In this tutorial, you will be importing an OpenAPI specification and creating a REST API proxy:
+In this tutorial, you will be providing a URL to a sample OpenAPI definition of a sample 
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
 
-2. On the **Home**** page, click the **+ Create Project**.
-   
-    ![Create project](../../assets/img/tutorials/rest-api/create-project.png){.cInlineImage-small}
+2. Click **+ Create Project**. Enter a unique name and a description for the project, and click **Create**.
 
-3. Enter a unique name and description for your project and click **Create**. 
+3. On the **REST API Proxy** card, click **Create**.
 
-    You can now start adding components to your project. Let's add a REST API proxy component.
+4. Click **Try with Sample URL** to create an API proxy for a sample API.
 
-    !!! tip
-        Once you create a project successfully, you are taken to the components landing page. You can also select the components icon from the left menu and open the **Components** landing page. 
+5. Enter API proxy details as follows:
 
-4. On the **Components** page, click **Create** under the **REST API Proxy** card.
-   
-5. Click **Import your OpenAPI**.
+    |  **Field**    | **Value**                                   |
+    |---------------|---------------------------------------------|
+    | **Name**      | `HR API`                                    |
+    | **Base Path** | `<DEFAULT_VALUE>/ abc-hr`                   |
+    | **Version**   | `1.0.0`                                     |
+    | **Target**    | `https://samples.choreoapps.dev/company/hr` |
 
-6. Select the **Input Type** and provide the OpenAPI URL as follows:
- 
-    | **Field** | **Value** |
-    |-----------------|-----------------------|
-    | **Input Type** | OpenAPI URL |
-    | **OpenAPI URL** | `https://petstore3.swagger.io/api/v3/openapi.json` |
+7. Click **Create**. 
+
+The API proxy opens on a separate page.
+
+## Step 2: Developing the REST API Proxy
+
+Let's develop the REST API proxy by adding an operation. To do this, follow the steps given below:
```

Comment:
Let's design? develop is generally implementing

Reply:
But the UI itself uses the word "Develop" for proxies.

Reply:
Ack